tests/Valit.Tests/String/String_MaxLength_Tests.cs

```diff
@@ -33,9 +33,9 @@ public void String_MaxLength_Returns_Proper_Result_For_Left_Value(int value, boo
         }
 
         [Theory]
-        [InlineData(0, false)]
-        [InlineData(4, false)]
-        [InlineData(8, false)]
+        [InlineData(0, true)]
+        [InlineData(4, true)]
+        [InlineData(8, true)]
```

Comment:
A string of zero length now considered to have a length less than or equal to 0, 4, or 8.